src/links/dto/create-link.dto.ts

```diff
@@ -1,12 +1,12 @@
-import { IsUrl, IsUUID } from 'class-validator';
+import { IsUrl } from 'class-validator';
 
 export class CreateLinkDto {
-  @IsUrl()
-  shortUrl: string;
-
-  @IsUrl()
+  @IsUrl({
```

Comment:
**praise**: Excellent use of `IsUrl` options to ensure URL validation is robust.